### PR TITLE
Simplify code by removing 2 global variables.

### PR DESCRIPTION
**Pull request by Yexo**

### CODE REVIEW

**Review by frosch123 — DISMISSED** [review]
The removal of the timer could be a third commit, if you want.

**Comment by Yexo** [comment]
> The removal of the timer could be a third commit, if you want.

Thanks. It's indeed cleaner this way.

File: src/industry_gui.cpp [hunk 1]
```diff
@@ -447,10 +448,14 @@ class BuildIndustryWindow : public Window {
 						strdim.width = max_minwidth;
 					}
 					d = maxdim(d, strdim);
+
+					if (indsp->grf_prop.grffile != nullptr) {
+						extra_lines_newgrf = max(extra_lines_newgrf, 4);
```
Comment (LordAro) [inline src/industry_gui.cpp]: Shouldn't this just be
```suggestion
						extra_lines_newgrf = 4;
```
? It's not modified anywhere else (also preferrably with some comment/constant name ? )

Comment (Yexo) [reply src/industry_gui.cpp]: Done, and added a comment.

**Review by LordAro — APPROVED** [review]
LGTM

**Pull request merged**